runtime/doc/options.txt

```diff
@@ -5780,6 +5781,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Name of the word list file where words are added for the |zg| and |zw|
 	commands.  It must end in ".{encoding}.add".  You need to include the
 	path, otherwise the file is placed in the current directory.
+	The path may include characters from 'isfname', space, comma and '@'.
```

Comment:
I think this needs https://github.com/vim/vim/commit/bc49c5f48f89c2d6f4d88ee77f44a11d68293be3